README.md

```diff
@@ -1,21 +1,98 @@
 
-The fence-agents-remediation (*FAR*) is a Kubernetes operator generated using the [operator-sdk](https://github.com/operator-framework/operator-sdk), and it is part of [Medik8s](https://github.com/medik8s) operators. This operator is designed to run an existing set of [upstream fencing agents](https://github.com/ClusterLabs/fence-agents) for environments with a traditional API end-point (e.g., [IPMI](https://en.wikipedia.org/wiki/Intelligent_Platform_Management_Interface)) for power cycling cluster nodes.
+fence-agents-remediation (*FAR*) is a Kubernetes operator that uses well-known agents to fence and remediate unhealthy nodes. The remediation includes rebooting the unhealthy node using a fence agent, and then evicting workloads from the unhealthy node. The operator is recommended when a node becomes unhealthy, and we want remediate it by completely isolating the node from a cluster and help with recovering its workload. Isolation is needed, since we can’t “trust” the unhealthy node, to prevent it from accessing the shared resources like [RWO volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes), and recovering the workloads helps to acclerate and keep their running time.
 
-The operator watches for new or deleted custom resources (CRs) called `FenceAgentsRemediation` (or `far`) which trigger a fence-agent to remediate a node, based on the CR's name.
-FAR operator was designed to run with the Node HealthCheck Operator [(NHC)](https://github.com/medik8s/node-healthcheck-operator) as external remediator for easier and smoother experience, but it can be used as a standalone remediation for the more advanced user.
-FAR joins Medik8s as another remediator alternative for NHC, apart from [Self Node Remediation](https://github.com/medik8s/self-node-remediation) and [Machine Deletion Remediation](https://github.com/medik8s/machine-deletion-remediation) which are also from the [Medik8s](https://www.medik8s.io/) group.
+FAR is one of the remediator operators by [Medik8s](https://www.medik8s.io/remediation/remediation/), such as [Self Node Remediation](https://github.com/medik8s/self-node-remediation) and [Machine Deletion Remediation](https://github.com/medik8s/machine-deletion-remediation), that were designed to run with the Node HealthCheck Operator [(NHC)](https://github.com/medik8s/node-healthcheck-operator) which detects an unhealthy node and creates remediation Custom Resource ([CR](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)). It is recommended to use FAR with NHC for an easier and smoother experience by fully automating the remediation process, but it can be used as a standalone remediator for the more experienced user. Moreover, like other Medik8s operators FAR was generated using the [operator-sdk](https://github.com/operator-framework/operator-sdk), and it supports Operator Lifecycle Manager ([OLM](https://olm.operatorframework.io/docs/)).
```

Comment:
Can you be more specific in this long line?

Reply:
>Also, we could benefit from breakdown long lines for the review and for reading later

I would love to do that but the output seems quite odd when I add open a new line for a new sentence in the paragraph
![image](https://github.com/medik8s/fence-agents-remediation/assets/19728420/451a452c-5477-43f5-8450-d2c39882600a)
But I guess it can be resolved I "block" these lines with a paragraph sign (something like `<p>` in HTML)

Reply:
uhm, I didn't expect that. Markdown should start a new paragraph only after an empty line :thinking:

Reply:
+1. Maybe I am missing something here

Reply:
How did you render that output from your screenshot? It's wrong for sure for line breaks without empty line...

Reply:
I have used the VSCode IDE render option by extension [Markdown Preview Enhanced](https://github.com/shd101wyy/vscode-markdown-preview-enhanced). I have verified that in https://markdownlivepreview.com/ and indeed my assumption was wrong as maybe Markdown Preview Enhanced has a bug.

Reply:
I was about to suggest opening an issue for that extension... but wow, 1.2k open issues does not look promising 🙈